identity/src/lib/db/models.rs

```diff
@@ -1,8 +1,9 @@
-use chrono::NaiveDateTime;
+use chrono::{NaiveDateTime, Utc};
 
 use super::schema::*;
+use crate::rpc::models as RpcModels;
```

Comment:
I'd prefer `rpc_models`, since it is a module and not a `trait`, `struct` or a `type`.
```suggestion
use crate::rpc::models as rpc_models;
```